Fortran/UnitTests/defined_io/list-read.f90

```diff
@@ -0,0 +1,65 @@
+module m
```

Comment:
Is this a verbatim copy of a test case in a bug report?  If so, please link to it in a source comment.